core/src/main/java/org/pentaho/platform/web/http/api/resources/services/ISchedulerServicePlugin.java

```diff
@@ -0,0 +1,111 @@
+/*!
+ * HITACHI VANTARA PROPRIETARY AND CONFIDENTIAL
```

Comment:
Header should of GPL, it seams.

Reply:
Can you restate your ask or course of action?

Reply:
This copyright header is used in files of EE projects, not CE.

Reply:
addressed via: https://github.com/pentaho/pentaho-scheduler-plugin/pull/60/commits/223802766d389eccdd3e76d6614bf03c1b625a3c